src/extensions/timeseries/timeseries.py

```diff
@@ -1,7 +1,12 @@
 import labbox_ephys as le
 
 
-@le.register_recording_view
-def Timeseries(*, sorting: le.LabboxEphysSortingExtractor, recording: le.LabboxEphysRecordingExtractor):
+@le.register_recording_view('Timeseries')
```

Comment:
The optional parameter of register_recording_view is the name of the plugin (how it will be called from python). If omitted it would be equal to the function name. But in this case we have two different plugins with the same name, so we use this argument.